app/components/blacklight/skip_link_item_component.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Blacklight
+  class SkipLinkItemComponent < Blacklight::Component
+    def initialize(text:, href:)
+      @text = text
```

Comment:
can we make `link_classes` an argument with a default value of `'d-inline-flex py-2 px-3'`?